packages/toolkit/tsup.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig({
+  entry: ['src/index.ts'],
+  format: ['esm'],
+  dts: {
+    entry: 'src/index.ts',
+    resolve: true,
+  },
+  treeshake: true,
+  clean: true,
+  esbuildOptions(options) {
+    options.define = { 'import.meta.vitest': 'undefined' }
```

Comment:
By the way, I've been meaning to ask you for a while. Why do some functions have tests for them placed in the same file? Why can't we just put them in separate files and remove that redundancy of having to replace `import.meta.vitest` with undefined?

Reply:
I just find it more convenient sometimes, especially when there's not a lot of code and not a lot of test cases. Having a separate `*.spec.ts` gets in the way of the file structure, so if I have a couple files, the duplicated names of test files are a bit annoying to deal with.